sqlglot/tokens.py

```diff
@@ -984,14 +986,20 @@ class Tokenizer(metaclass=_Tokenizer):
         "_peek",
         "_prev_token_line",
         "_rs_dialect_settings",
+        "_use_rs_tokenizer",
     )
 
-    def __init__(self, dialect: DialectType = None) -> None:
+    def __init__(
+        self, dialect: DialectType = None, USE_RS_TOKENIZER: t.Optional[bool] = None
```

Comment:
Only constants should be capitalized:
```suggestion
        self, dialect: DialectType = None, use_rs_tokenizer: t.Optional[bool] = None
```